panther_driver/src/panther_hardware.py

```diff
@@ -93,6 +100,26 @@ def _setup_gpio() -> None:
         GPIO.setup(DRIVER_EN, GPIO.OUT, initial=0)
         GPIO.setup(E_STOP_RESET, GPIO.IN)  # USED AS I/O
 
+    @staticmethod
+    def _shutdown_host() -> None:
+        # EXAMPLE: https://www.linode.com/docs/guides/use-paramiko-python-to-ssh-into-a-server/
+        host = "10.15.20.2"
+        username = "husarion"
+
+        pkey = paramiko.RSAKey.from_private_key_file("/root/.ssh/id_rsa")
+        client = paramiko.SSHClient()
+        policy = paramiko.AutoAddPolicy()
+        client.set_missing_host_key_policy(policy)
+        client.connect(host, username=username, pkey=pkey)
+        _stdin, _stdout, _stderr = client.exec_command("sudo shutdown now")
+        print(_stdout.read().decode())
+        client.close()
+
+    def soft_stop(self):
+        button = Button(SHDN_INIT, pull_up=False)
+        button.wait_for_press()
+        self._shutdown_host()
+
     def _motor_start_sequence(self) -> None:
         """
         First start sequence for motors which is meant to power up modules in correct order
```

Comment:
`_motor_start_sequence` should be just under `_setup_gpio` as it is called in constructor

Reply:
:+1: